helm/hpcc/templates/roxie.yaml

```diff
@@ -168,7 +168,7 @@ spec:
 ---
 {{- range $service := $roxie.services }}
 {{- if ne (int $service.servicePort)  0 }}
-{{ include "hpcc.addService" ( dict "root" $ "name" $service.name "service" $service "selector" $servername "defaultPort" $service.servicePort ) }}
+{{ include "hpcc.addService" ( dict "root" $ "name" $service.name "service" $service "selector" $servername "defaultPort" $service.servicePort "appProtocolHTTP" true) }}
```

Comment:
Are all roxie services http?

Reply:
Yes, because roxie supports both it's native protocol (do we have a name for that?) and http on the same port.  So the load balancer can use http for it's check.